lib/galaxy/dependencies/pipfiles/update.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+
+commit=0
+
+usage() {
+cat << EOF
+Usage: ${0##*/} [-c]
+
+Use pipenv to regenerate locked and hashed versions of Galaxy dependencies.
+Use -c to automatically commit these changes (be sure you have no staged git
+changes).
+
+EOF
+}
+
+while getopts ":i" opt; do
```

Comment:
I think that `":i"` should be changed to `"hc"` here.

Reply:
Ahh - that makes sense - copy/paste mistake.